src/app/interfaces/NFT/NFT.ts

```diff
@@ -2,6 +2,8 @@ export interface NFT {
   id: string;
   name: string;
   ftx_id: string;
+  attributes: any;
```

Comment:
can we try to type this?

Reply:
We could do that for herolete, but the new tokens for hall of fame wont be the same and would lose point.